e2e-cypress/Jenkinsfile.template

```diff
@@ -55,7 +45,26 @@ odsComponentPipeline(
       ]
     )
   }
-
+  if (fileExists('cypress/screenshots.zip')) {
```

Comment:
Maybe I'm missing something, but right now it seems that videos will go to a zip and nowhere else from there. Maybe there should be a block like the following as well:
```
  if (fileExists('cypress/videos.zip')) {
    odsComponentStageUploadToNexus(context,
      [
        distributionFile: 'cypress/videos.zip',
        repository: 'leva-documentation',
        repositoryType: 'raw',
        targetDirectory: "${targetDirectory}"
      ]
    )
  }
```

However, I don't see the need for videos being part of any documentation, if there are things like test protocols and screenshots already available. I still believe that it might be a good option to keep videos as build artifacts with Jenkins. Maybe we bring back the block
```
      if (fileExists('cypress/videos')) {
        zip zipFile: 'cypress/videos.zip', archive: false, dir: 'cypress/videos'
        stash(name: "acceptance-test-videos-${context.componentId}-${context.buildNumber}", includes: 'cypress/videos.zip', allowEmpty: true)
        archiveArtifacts artifacts: 'cypress/videos.zip', fingerprint: true, daysToKeep: 2, numToKeep: 3
      }
```
and set the `numToKeep` to 1 or leave it commented out. And we could add a warning as code comment that storage usage should be monitored when dealing with videos. Or we add a check on the file size against an upper limit before archiving the artifact.